pwndbg/color/syntax_highlight.py

```diff
@@ -0,0 +1,67 @@
+from __future__ import absolute_import
```

Comment:
new files should explicitly declare python encoding, we do this in every other file
```# -*- coding: utf-8 -*-```
https://www.python.org/dev/peps/pep-0263/

Reply:
Added